distribution/docs/src/main/resources/_contents/_draft-registry-contents/managing-registry-contents.adoc

```diff
@@ -0,0 +1,363 @@
+|String
+|yes
+
+|Contact Last Name
+|Contact Last Name
+|String
+|yes
+
+|Address
+|Address for listed contact
+|String
+|minimum one
+
+|Phone number
+|Contact phone number
+|
+|minimum one
+
+|Email
+|Contact email
+|String
+|minimum one
+
+|Custom Fields
+a|click *Add* button to add custom fields
+|Configurable
+|no
+
+|Associations
+a|click *Add* button to add associations
+|Configurable
+|no
+
+|===
+
+.Collections (Click *Add* to add Content Collection(s))
+[cols="1,3,1,1" options="header"]
+|===
+|Field
+|Description
+|Type
+|Required
+
+|Content Name
+|Name for this metadata content
+|string
+|yes
+
+|Content Description
+|Short description for this metadata content
+|string
+|no
+
+|Content Object Type
+|The kind of content object this will be. Default value should be used in most cases.
+|string
+|yes
+
+|Custom Fields
+a|click *Add* button to add custom fields
+|Configurable
+|no
+
+|Associations
+a|click *Add* button to add associations
+|Configurable
+|no
+
+|===
+
+===== Adding a Service Binding to a Node
+
+Advertise the methods other nodes use to connect to the local ${branding} instance.
+
+* Navigate to ${admin-console}.
+* Select ${ddf-registry} or ${ddf-catalog}.
+** (*Local Registry Nodes* tab is editable from either application.)
+* Click the name of the desired node.
+* Click the *Services* tab.
+* Click *Add* to add a service.
+* Expand new Service.
+* Enter Service name and details.
+* Click *Add* to add binding.
+* Select Service Binding type.
+** Select one of the defaults or _empty_ for a custom service binding.
+** If selecting _empty_, fill in all required fields.
+* Click Save.
+
+==== Publishing to Others
+
+Send details about the local ${branding} instance to other nodes.
+
+* Navigate to the *Remote Registries* tab in either ${ddf-registry} or ${ddf-catalog} application.
+* Click *Add* to add a remote registry.
+* Give node a unique local name.
+* Enter Registry Service (CSW) Url.
+* Confirm *Allow Push* is checked.
+* Click *Add* to save the changes.
+* Navigate to the *Sources* Tab in ${ddf-catalog} App
+* Click desired node to be published.
+* Under *Operations*, click the *Publish to ... * link that corresponds to the desired registry.
+
+==== Subscribing to Another Node
+
+Receive details about another node.
+
+* Navigate to the *Remote Registries* tab in either ${ddf-registry} or ${ddf-catalog} application.
+* Click *Add* to add a remote registry.
+* Give new node a unique Local name.
+* Add the URL to access node.
+* Enter any needed credentials in the Username/password fields.
+* Click *Save*/*Add*.
+
+===== Editing a Subscription
+
+Update the configuration of an existing subscription.
+
+* Navigate to the *Remote Registries* tab in either ${ddf-registry} or ${ddf-catalog} application.
+* Click the name of the desired subscription.
+* Make changes.
+* Click *Save*.
+
+===== Deleting a Subscription
+
+Remove a subscription.
+
+* Click the *Delete* icon corresponding to the desired node to delete.
```

Comment:
* Click the *Delete* icon at the top of the Remote Registries tab.
* Check the boxes of the Registry Nodes to *Delete* and then select the Delete button.